src/policy/mod.rs

```diff
@@ -1,3 +1,16 @@
+//! Memory policies that can be used for spaces.
+
+/// This class defines and manages spaces.  Each policy is an instance
```

Comment:
Each policy is ...

to:

Each space is an instance of a policy?

Reply:
Correct. That was copied from Java MMTk, and it is incorrect.